### ISSUE TITLE
should be able to separate out dictionary's or supply your own

### ISSUE THREAD

**Issue by mog13** [mog13/Splain#7]
Rather than one universal dictionary it would be useful to be able to specify one. This way different sources/characters can operate from there own dictionary.

**Comment by mog13** [comment]
This might not be very clear but after chatting with @Luke-Rogers a use case might be for something like languages. You could have a dictionary for English and then separate ones for other languages. This way your templates wouldn't need to changes but you could pass in which dictionary to use. (if none is specified it would default). I'm imagining something like this (but am open to suggestions): 

```JS
Splain.addEntry({greetings:["hi","hello"]},"English");
Splain.addEntry({greetings:["bonjour","salut"]},"French");

let curLanguage = "English";

console.log(Splain.process("{{greetings}}",{},curLanguage));
```

**Comment by mog13** [comment]
closed via #69
